function f(Exception $e): void {
    throw $e;
}
                